export const genotypes_view_test_consequences_data = [
    {
        "label": "start retained variant",
        "description": "A sequence variant where at least one base in the start codon is changed, but the start remains",
        "SO_term": "start_retained_variant",
        "SO_accession": "SO:0002019",
        "consequence_ranking": "15"
    },
    {
        "label": "non coding transcript exon variant",
        "description": "A sequence variant that changes non-coding exon sequence in a non-coding transcript",
        "SO_term": "non_coding_transcript_exon_variant",
        "SO_accession": "SO:0001792",
        "consequence_ranking": "20"
    },
    {
        "SO_term": "non_coding_transcript_variant",
        "description": "A transcript variant of a non coding RNA gene",
        "label": "non coding transcript variant",
        "consequence_ranking": "23",
        "SO_accession": "SO:0001619"
    },
    {
        "description": "A sequence variant located within a transcription factor binding site",
        "SO_term": "TF_binding_site_variant",
        "label": "TF binding site",
        "consequence_ranking": "30",
        "SO_accession": "SO:0001782"
    },
    {
        "SO_accession": "SO:0002012",
        "consequence_ranking": "7",
        "label": "start lost",
        "SO_term": "start_lost",
        "description": "A codon variant that changes at least one base of the canonical start codon"
    },
    {
        "SO_accession": "SO:0001587",
        "consequence_ranking": "4",
        "label": "stop gained",
        "SO_term": "stop_gained",
        "description": "A sequence variant whereby at least one base of a codon is changed, resulting in a premature stop codon, leading to a shortened transcript"
    },
    {
        "SO_accession": "SO:0001889",
        "consequence_ranking": "8",
        "label": "transcript amplification",
        "SO_term": "transcript_amplification",
        "description": "A feature amplification of a region containing a transcript"
    },
    {
        "consequence_ranking": "3",
        "SO_accession": "SO:0001574",
        "description": "A splice variant that changes the 2 base region at the 3' end of an intron",
        "SO_term": "splice_acceptor_variant",
        "label": "splice acceptor variant"
    },
    {
        "SO_accession": "SO:0001822",
        "consequence_ranking": "11",
        "label": "inframe deletion",
        "description": "An inframe non synonymous variant that deletes bases from the coding sequence",
        "SO_term": "inframe_deletion"
    },
    {
        "SO_accession": "SO:0001891",
        "consequence_ranking": "33",
        "label": "regulatory region amplification",
        "description": "A feature amplification of a region containing a regulatory region",
        "SO_term": "regulatory_region_amplification"
    },
    {
        "label": "regulatory region ablation",
        "SO_term": "regulatory_region_ablation",
        "description": "A feature ablation whereby the deleted region includes a regulatory region",
        "SO_accession": "SO:0001894",
        "consequence_ranking": "31"
    },
    {
        "SO_accession": "SO:0001907",
        "consequence_ranking": "36",
        "label": "feature elongation",
        "description": "A sequence variant that causes the extension of a genomic feature, with regard to the reference sequence",
        "SO_term": "feature_elongation"
    },
    {
        "consequence_ranking": "14",
        "SO_accession": "SO:0002170",
        "description": "A sequence variant that falls in the region between the 3rd and 6th base after splice junction (5' end of intron).",
        "SO_term": "splice_donor_region_variant",
        "label": "splice donor region variant"
    },
    {
        "SO_accession": "SO:0001620",
        "consequence_ranking": "17",
        "label": "mature miRNA variant",
        "description": "A transcript variant located with the sequence of the mature miRNA",
        "SO_term": "mature_miRNA_variant"
    },
    {
        "SO_accession": "SO:0001575",
        "consequence_ranking": "3",
        "label": "splice donor variant",
        "SO_term": "splice_donor_variant",
        "description": "A splice variant that changes the 2 base region at the 5' end of an intron"
    },
    {
        "label": "3 prime UTR variant",
        "description": "A UTR variant of the 3' UTR",
        "SO_term": "3_prime_UTR_variant",
        "SO_accession": "SO:0001624",
        "consequence_ranking": "19"
    },
    {
        "label": "coding sequence variant",
        "SO_term": "coding_sequence_variant",
        "description": "A sequence variant that changes the coding sequence",
        "SO_accession": "SO:0001580",
        "consequence_ranking": "16"
    },
    {
        "label": "inframe insertion",
        "SO_term": "inframe_insertion",
        "description": "An inframe non synonymous variant that inserts bases into in the coding sequence",
        "SO_accession": "SO:0001821",
        "consequence_ranking": "10"
    },
    {
        "consequence_ranking": "21",
        "SO_accession": "SO:0001627",
        "SO_term": "intron_variant",
        "description": "A transcript variant occurring within an intron",
        "label": "intron variant"
    },
    {
        "description": "A variant in a transcript that is the target of NMD",
        "SO_term": "NMD_transcript_variant",
        "label": "NMD transcript variant",
        "consequence_ranking": "22",
        "SO_accession": "SO:0001621"
    },
    {
        "consequence_ranking": "25",
        "SO_accession": "SO:0001632",
        "SO_term": "downstream_gene_variant",
        "description": "A sequence variant located 3' of a gene",
        "label": "downstream gene variant"
    },
    {
        "consequence_ranking": "12",
        "SO_accession": "SO:0001583",
        "SO_term": "missense_variant",
        "description": "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved",
        "label": "missense variant"
    },
    {
        "label": "splice polypyrimidine tract variant",
        "description": "A sequence variant that falls in the polypyrimidine tract at 3' end of intron between 17 and 3 bases from the end (acceptor -3 to acceptor -17)",
        "SO_term": "splice_polypyrimidine_tract_variant",
        "SO_accession": "SO:0002169",
        "consequence_ranking": "13"
    },
    {
        "consequence_ranking": "15",
        "SO_accession": "SO:0001819",
        "SO_term": "synonymous_variant",
        "description": "A sequence variant where there is no resulting change to the encoded amino acid",
        "label": "synonymous variant"
    },
    {
        "consequence_ranking": "38",
        "SO_accession": "SO:0001628",
        "SO_term": "intergenic_variant",
        "description": "A sequence variant located in the intergenic region, between genes",
        "label": "intergenic variant"
    },
    {
        "label": "splice donor 5th base variant",
        "SO_term": "splice_donor_5th_base_variant",
        "description": "A sequence variant that causes a change at the 5th base pair after the start of the intron in the orientation of the transcript",
        "SO_accession": "SO:0001787",
        "consequence_ranking": "4"
    },
    {
        "consequence_ranking": "37",
        "SO_accession": "SO:0001906",
        "description": "A sequence variant that causes the reduction of a genomic feature, with regard to the reference sequence",
        "SO_term": "feature_truncation",
        "label": "feature truncation"
    },
    {
        "consequence_ranking": "14",
        "SO_accession": "SO:0001626",
        "SO_term": "incomplete_terminal_codon_variant",
        "description": "A sequence variant where at least one base of the final codon of an incompletely annotated transcript is changed",
        "label": "incomplete terminal codon variant"
    },
    {
        "consequence_ranking": "5",
        "SO_accession": "SO:0001589",
        "description": "A sequence variant which causes a disruption of the translational reading frame, because the number of nucleotides inserted or deleted is not a multiple of three",
        "SO_term": "frameshift_variant",
        "label": "frameshift variant"
    },
    {
        "description": "A sequence variant where at least one base of the terminator codon (stop) is changed, resulting in an elongated transcript",
        "SO_term": "stop_lost",
        "label": "stop lost",
        "consequence_ranking": "6",
        "SO_accession": "SO:0001578"
    },
    {
        "SO_accession": "SO:0001623",
        "consequence_ranking": "18",
        "label": "5 prime UTR variant",
        "SO_term": "5_prime_UTR_variant",
        "description": "A UTR variant of the 5' UTR"
    },
    {
        "consequence_ranking": "13",
        "SO_accession": "SO:0001630",
        "SO_term": "splice_region_variant",
        "description": "A sequence variant in which a change has occurred within the region of the splice site, either within 1-3 bases of the exon or 3-8 bases of the intron",
        "label": "splice region variant"
    },
    {
        "label": "regulatory region variant",
        "description": "A sequence variant located within a regulatory region",
        "SO_term": "regulatory_region_variant",
        "SO_accession": "SO:0001566",
        "consequence_ranking": "36"
    },
    {
        "SO_accession": "SO:0001895",
        "consequence_ranking": "26",
        "label": "TFBS ablation",
        "description": "A feature ablation whereby the deleted region includes a transcription factor binding site",
        "SO_term": "TFBS_ablation"
    },
    {
        "consequence_ranking": "24",
        "SO_accession": "SO:0001631",
        "SO_term": "upstream_gene_variant",
        "description": "A sequence variant located 5' of a gene",
        "label": "upstream gene variant"
    },
    {
        "consequence_ranking": "28",
        "SO_accession": "SO:0001892",
        "SO_term": "TFBS_amplification",
        "description": "A feature amplification of a region containing a transcription factor binding site",
        "label": "TFBS amplification"
    },
    {
        "label": "transcript ablation",
        "SO_term": "transcript_ablation",
        "description": "A feature ablation whereby the deleted region includes a transcript feature",
        "SO_accession": "SO:0001893",
        "consequence_ranking": "1"
    },
    {
        "description": "A sequence variant where at least one base in the terminator codon is changed, but the terminator remains",
        "SO_term": "stop_retained_variant",
        "label": "stop retained variant",
        "consequence_ranking": "15",
        "SO_accession": "SO:0001567"
    },
    {
        "SO_term": "sequence_variant",
        "description": "A sequence_variant is a non exact copy of a sequence_feature or genome exhibiting one or more sequence_alteration",
        "label": "sequence variant",
        "consequence_ranking": "39",
        "SO_accession": "SO:0001060"
    },
    {
        "description": "A sequence_variant which is predicted to change the protein encoded in the coding sequence",
        "SO_term": "protein_altering_variant",
        "label": "protein altering variant",
        "consequence_ranking": "12",
        "SO_accession": "SO:0001818"
    }
]